examples/flat-serialisation/no-notifiable-owner.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
As above. This example should probably be removed as it does not match option 1 from #57